libvast/src/detail/string.cpp

```diff
@@ -153,6 +153,15 @@ std::vector<std::string_view> split(std::string_view str, std::string_view sep,
   return pos;
 }
 
+std::string_view trim(std::string_view str) {
+  auto not_space = [](char c) {
```

Comment:
```suggestion
  auto not_space = [](unsigned char c) {
```

Actually UB otherwise, see https://en.cppreference.com/w/cpp/string/byte/isspace

> The behavior is undefined if the value of ch is not representable as `unsigned char` and is not equal to [EOF](http://en.cppreference.com/w/cpp/io/c).